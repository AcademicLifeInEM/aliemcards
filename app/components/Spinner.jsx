import React from 'react';

const Loader = ({ visible }) =>
  <div className={visible ? 'spinner' : 'spinner spinner-hide'}>
    <i className="material-icons">autorenew</i>
  </div>;

Loader.propTypes = {
  visible: React.PropTypes.bool,
};

export default Loader;
