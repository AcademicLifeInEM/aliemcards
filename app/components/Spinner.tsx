import * as React from 'react';

export default () =>
  <div className="spinner">
    <i className="material-icons">autorenew</i>
  </div>;
