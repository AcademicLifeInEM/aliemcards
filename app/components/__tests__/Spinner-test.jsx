import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../Spinner';

describe('<Spinner />', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <Spinner />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
