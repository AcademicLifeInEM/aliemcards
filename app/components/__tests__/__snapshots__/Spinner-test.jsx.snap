// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render correctly 1`] = `
<div
  className="spinner"
>
  <i
    className="material-icons"
  >
    autorenew
  </i>
</div>
`;
